app/Models/Building.php

```diff
@@ -538,33 +543,51 @@ public function getBuildingElement($short, InputSource $inputSource = null)
     public function getBuildingElementsForMe($short)
     {
         return $this->buildingElements()
-                    ->withoutGlobalScope(GetValueScope::class)
-                    ->leftJoin(
-                        'elements as e',
-                        'building_elements.element_id',
-                        '=',
-                        'e.id'
-                    )
-                    ->where('e.short', $short)->select(['building_elements.*']
+            ->withoutGlobalScope(GetValueScope::class)
+            ->leftJoin(
+                'elements as e',
+                'building_elements.element_id',
+                '=',
+                'e.id'
+            )
+            ->where('e.short', $short)->select(['building_elements.*']
             )->get();
     }
 
     /**
-     * @param  string  $short
+     * @param string $short
      *
      * @return BuildingService|null
      */
     public function getBuildingService($short, InputSource $inputSource)
     {
         return $this->buildingServices()
-                    ->forInputSource($inputSource)
-                    ->leftJoin(
-                        'services as s',
-                        'building_services.service_id',
-                        '=',
-                        's.id'
-                    )
-                    ->where('s.short', $short)->first(['building_services.*']);
+            ->forInputSource($inputSource)
+            ->leftJoin(
+                'services as s',
+                'building_services.service_id',
+                '=',
+                's.id'
+            )
+            ->where('s.short', $short)->first(['building_services.*']);
+    }
+
+    /**
+     * @param string $short
+     *
+     * @return ServiceValue|null
+     */
+    public function getServiceValue($short, InputSource $inputSource)
```

Comment:
I'm pretty sure this was removed in a previous PR as to being unused. @frogeyedman Please confirm if this should stay or not